### PR TITLE
Added github action to add and build all modules in index

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

File: .github/workflows/all-modules-installable.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+      # Add masterfiles
+      - name: Add masterfiles
+        run: cfbs --index index.json add masterfiles
+
+      # Add all module
+      - name: Add all modules
+        run: |
+          chmod +x /home/runner/work/build-index/build-index/.github/workflows/add-all-modules.sh
+          /home/runner/work/build-index/build-index/.github/workflows/add-all-modules.sh
```
Comment (larsewi) [inline .github/workflows/all-modules-installable.yaml:48]: I don't think you need absolute path. I think `.github/workflows/add-all-modules.sh` should do just fine.

File: .github/workflows/add-all-modules.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/env bash
+
+cfbs search --index index.json | \
+    awk '{print $1}' | \
+    xargs -n1 cfbs --index index.json add
```
Comment (larsewi) [inline .github/workflows/add-all-modules.sh:5]: GHA did not like pipes?

**Review by larsewi — APPROVED** [review]
LGTM

File: .github/workflows/add-all-modules.sh (hunk 2 above)
Comment (nickanderson) [reply .github/workflows/add-all-modules.sh:5]: mm lets see

File: .github/workflows/all-modules-installable.yaml (hunk 1 above)
Comment (nickanderson) [reply .github/workflows/all-modules-installable.yaml:48]: ack, will try

File: .github/workflows/add-all-modules.sh (hunk 2 above)
Comment (nickanderson) [reply .github/workflows/add-all-modules.sh]: it liked pipes just fine.

**Pull request merged**